backend/src/event/event.controller.ts

```diff
@@ -34,6 +34,20 @@ export class EventController {
     return this.eventService.getAllEvents();
   }
 
+  @Get(EventController.SORTED_EQUIPMENTS_API_PATH)
```

Comment:
had to move the endpoints because otherwise, they would get overridden by the `EVENT_ID_API_PATH` route